### PR TITLE
INTENG-3494 The aasa validator with domain results prefilled.

### PR DESCRIPTION
**Pull request by amruta-d**
Description of the change
https://branch.atlassian.net/browse/INTENG-3494
The aasa validator will now show the domain results if a query parameter 'domain' is appended to the URL. e.g. https://branch.io/resources/aasa-validator/?domain=branchster.app.link

Test Plan
Tested locally.

Reviewers
@aaustin

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "server": {
-        "port": 80
+        "port": 8000
```
Comment (aaustin) [inline config.json]: not sure if this will break production but you probably don't want to leave this change in there

**Comment by aaustin** [comment]
@rubinsingh I don't know much about this code - can you take a gander?

File: README.md [hunk 2]
```diff
@@ -37,3 +37,18 @@ The original tool used validation rules that were more restrictive. Branch's too
...
+1. Install nodejs version 4.2.4 (use npm install n -g to manage multiple node version)
+2. Install npm modules: npm install
+
+### Running it
+
+1. Cloning the repository
+
+2. `cd` into the repository
+
+3. Run `npm start` or `node index.js`
```
Comment (rubinsingh) [inline README.md]: We should add a point about changing the port in config.json to 8000

File: static/index.html [hunk 3]
```diff
@@ -30,7 +30,7 @@
     <script src="static/js/branchAttributionScripts.js"></script>
   </head>
 
-  <body ng-controller="MainController">
+  <body ng-controller="MainController" ng-init="init()">
```
Comment (rubinsingh) [inline static/index.html]: You could potentially remove ng-init="init()" and call $scope.init(); at the bottom of MainController.js.

**Comment by rubinsingh** [comment]
@amruta-d can you update README.md with instructions on how to change the port in config.json as well?

**Comment by amruta-d** [comment]
@rubinsingh Updated the Readme file with the port number changes.

**Comment by rubinsingh** [comment]
👍

**Pull request merged**